queries/authzed/highlights.scm

```diff
@@ -0,0 +1,38 @@
+((identifier) @keyword
+ (#any-of? @keyword "definition" "permission" "relation"))
+
+(permission_literal) @variable.builtin
+
+(permission (identifier) @type)
+(relation (identifier) @constant)
+(perm_expression (identifier) @property)
+
+[
+  (block_start)
+  (block_end)
+] @punctuation.bracket
+
+(block (identifier) (identifier) @constructor)
+
+[
+  (plus_literal)
+  (pipe_literal)
+] @operator
+
+(relation_literal) @function
+(rel_expression (identifier) @property)
+
+(relation
+  (rel_expression
+    ((hash_literal)
+     . (identifier) @constant) @comment))
```

Comment:
I have yet to recognize this, but what nodes are captured by this `@comment`? Is it `(hash_literal)`, or `(identifier)`, or both ?

If it's only `hash_literal`, something like this is better:
```suggestion
(hash_literal) @comment
.
(identifier) @constant))
```

Reply:
I believe the intention is to consider the has literal and subsequent identifier to both be part of the comment.

It's worth mentioning that after I apply the refactoring changes required by the upstream repository, this particular pattern will not be here anymore, so we may not want to get to the bottom of it 😄

Reply:
Looking forward to the updated queries